Src/ILGPU/Runtime/Cuda/CudaAccelerator.cs

```diff
@@ -482,6 +482,24 @@ public long GetFreeMemory()
             where T : unmanaged =>
             Allocate2DPitchedY<T>(extent, PitchedAllocationAlignmentInBytes);
 
+        /// <summary>
+        /// Maps a buffer allocated by another process of the specified size in bytes on this accelerator.
+        /// </summary>
+        /// <param name="ipcMemHandle">A IPC memory handle from another process.</param>
+        /// <param name="length">The number of elements to allocate.</param>
+        /// <param name="elementSize">The size of a single element in bytes.</param>
+        /// <returns>A mapped buffer of shared memory on this accelerator.</returns>
+        public CudaIpcMemoryBuffer MapRaw(CudaIpcMemHandle ipcMemHandle, long length, int elementSize)
+        {
+            if (length < 0)
+                throw new ArgumentOutOfRangeException(nameof(length));
+            if (elementSize < 1)
+                throw new ArgumentOutOfRangeException(nameof(elementSize));
+
+            Bind();
+            return new CudaIpcMemoryBuffer(this, ipcMemHandle, length, elementSize);
```

Comment:
QUESTION: Should there be a `flags` parameter in the method signature, which then gets passed here?

Reply:
Sorry, I went from top to bottom and only saw this later. Feel free to reply in either thread.
> What I'm not sure about is, whether we should even add them to the "convenience" methods. The currently only flag LazyEnablePeerAccess will enable peer access if the underlying memory is allocated on a different device than the device used by the current context.
> Obviously I'm quite new to this library, but my gut says this would be a rather niche case not worth complicating the API for.